compatibility.bs

```diff
@@ -1055,11 +1057,61 @@ On mobile platforms, including smaller iPad form factors
   </tbody>
 </table>
 
+
+<h2 id="removal">Deprecation and Removal</h2>
+
+<em>This section is non-normative.</em>
+
+While the web platform stives for backwards-compatibility, there are occasions which require the
+outright removal or discouraged use of some features. To support making these changes
+in the least disruptive and most consistent way for developers, we document them here. This is most
+beneficial as a decalration of intent by browser engines to act together in a well reasoned way.
+
+<h3 id="removal-features">Feature List</h3>
+
+<table>
+  <thead>
+    <tr>
+      <th><dfn for="deprecated feature list">Feature</dfn></th>
+      <th><dfn for="deprecated feature list">Deprecated from</dfn></th>
+      <th><dfn for="deprecated feature list">Removal</dfn></th>
+      <th><dfn for="deprecated feature list">Replacement</dfn></th>
+      <th><dfn for="deprecated feature list">Justification</dfn></th>
+      <th><dfn for="deprecated feature list">Automated Tools</dfn></th>
+    <tr>
+  </thead>
+  <tbody>
+    <tr>
+    </tr>
+  </tbody>
+</table>
+
+<h4 id="removal-features-process">Process</h4>
+
+Anyone can make an addition to the list by editing this specification.
+However, because some content describes the future plans of browser engines, the author can use
```

Comment:
This suggests we might end up with a bunch of engine/product specific deprecations... is that what we want? Or are we trying to cover deprecations/removals that fit the spirit of WHATWG, where there is alignment of at least 2 engines?

If we do want to cover engine-specific stuff, maybe a separate table for that? Experimental deprecations or something.

Reply:
I think there are two relevant ideas here:

1. This is meant to be stuff in the spirit of WHATWG.
2. Since it has concrete stuff about the browser's future plans and anyone can write a PR we probably want to be careful about getting all the right contexts in reviewing it. I gated that on the editor's discretion.

Reply:
A potential path forward would be to move towards a general practice for folks to file issues against the spec to add deprecations, use those as a forum for alignment between engines, and land them in the spec when there's broader agreement on the direction.

If we want a more holistic list with a lower bar for entry, perhaps a wiki page or something similarly unofficial and light-weight would be appropriate?

Reply:
> A potential path forward would be to move towards a general practice for folks to file issues against the spec to add deprecations, use those as a forum for alignment between engines, and land them in the spec when there's broader agreement on the direction.

I think this is a good idea, and I'd be happy to make the contents of this document represent consensus, with PRs being a good place for discussion.